packages/blockchain-wallet-v4-frontend/src/components/Alerts/messages.tsx

```diff
@@ -1013,8 +1013,8 @@ const getAlertContent = (message, data = undefined) => {
     case C.WALLET_UPGRADE_ERROR:
       return buildMessageTemplate(
         <FormattedMessage
-          id='components.alerts.wallet_upgrade_error'
-          defaultMessage='Failed to upgrade to HD and save wallet.'
+          id='components.alerts.account_upgrade_error'
```

Comment:
@cmonti-bc you haven't changed the key for this so it will fail.

Reply:
great catch!